lib/app_profiler.rb

```diff
@@ -53,8 +55,10 @@ module Viewer
   mattr_reader   :profile_url_formatter,
     default: DefaultProfileFormatter
 
+  mattr_accessor :gecko_viewer_package, default: "https://github.com/tenderlove/profiler#v0.0.2"
   mattr_accessor :storage, default: Storage::FileStorage
-  mattr_accessor :viewer, default: Viewer::SpeedscopeViewer
+  mattr_accessor :viewer, default: Viewer::SpeedscopeViewer # DEPRECATED
+  mattr_accessor :speedscope_viewer, default: Viewer::SpeedscopeViewer
```

Comment:
```suggestion
  mattr_accessor :stackprof_viewer, default: Viewer::SpeedscopeViewer
```
This distinction should be per backend, not per viewer type